pkg/boot/fit/testdata/create_dummy_fit_image.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
```

Comment:
We don't use bash script in u-root as it creates too many dependencies for systems that only have u-root in source mode. Could you redo this as a Go function, that does not call Unix commands (i.e. there should be no exec.Command) and just have it create a temporary file? Or, further, have a pre-declared []byte in Go source that is the file? How big is this file when it is created?